### PR TITLE
Padding before title

### LINKED ISSUE #1614: Page nav title: add padding before the title
Add a bit of padding here by default?
image

### PR DESCRIPTION
**Pull request by MorningLit**
**What is the purpose of this pull request?**

- [ ] Documentation update
- [x] Bug fix
- [ ] Feature addition or enhancement
- [ ] Code maintenance
- [ ] Others, please explain:

Fixes #1614

**Overview of changes:**
Hi dev team, I am sending this PR here, as told by Prof Damith, because I am unsure if my fix is the correct approach. Do let me know what did I do wrong and where can I improve it!

Added a padding-left of 0.3125 rem which is the same as the element's padding-top and padding-bottom.
image

**Anything you'd like to highlight / discuss:**

**Testing instructions:**

**Proposed commit message: (wrap lines at 72 characters)**
Add left padding to page nav title

---

**Checklist:** :ballot_box_with_check:

- [ ] Updated the documentation for feature additions and enhancements
- [ ] Added tests for bug fixes or features
- [x] Linked all related issues
- [x] No blatantly unrelated changes
- [ ] Pinged someone for a review!

### CODE REVIEW

**Review by jonahtanjz — COMMENTED** [review]
Thanks for working on this @MorningLit 👍 

A few suggestions:

Wonder if it is better to add it [here](https://github.com/MarkBind/markbind/blob/master/packages/core-web/src/styles/page-nav.css#L22) instead so that it will be applied to current markbind sites as well. Adding it inside `core/template` will only affect newly `init` sites. 

`0.3125rem` still seems a little tight, maybe can try out `0.5rem`?

You may also want to run `npm run lintfix` and then `npm run test` first before pushing to lint and ensure all tests are passing.

**Comment by MorningLit** [comment]
Appreciate the feedback @jonahtanjz !
I have followed your feedback and made changes accordingly, do let me know if there is anything else I need to improve on!

File: packages/core-web/src/styles/page-nav.css [hunk 1]
```diff
@@ -24,6 +24,10 @@
     white-space: inherit;
 }
 
+#page-nav > .nav-component > .navbar-brand.page-nav-title {
+    padding-left: 0.5rem;
```
Comment (jonahtanjz) [inline packages/core-web/src/styles/page-nav.css:28]: How about shifting this into the existing `.page-nav-title` block? Shouldn't require the first few selectors as there should only be one page-nav on every page.

**Review by jonahtanjz — COMMENTED** [review]
Thanks for updating 👍 

Perhaps your commit message can be a little more specific? Maybe something like `Add left padding to page nav title`?

**Comment by MorningLit** [comment]
Updated my PR based on your comments 👍
Anything else this PR needs to improve on?

**Review by jonahtanjz — APPROVED** [review]
LGTM 👍 Thanks again for working on this @MorningLit!

**Pull request merged**